src/main/java/marquez/db/DatasetDao.java

```diff
@@ -41,8 +41,7 @@
 
   @SqlUpdate(
       "INSERT INTO datasets (guid, namespace_guid, datasource_uuid, urn, description, name) "
-          + "VALUES (:uuid, :namespaceUuid, :datasourceUuid, :urn, :description, :name) "
-          + "ON CONFLICT (urn) DO NOTHING")
+          + "VALUES (:uuid, :namespaceUuid, :datasourceUuid, :urn, :description, :name) ")
```

Comment:
Minor: You can remove the trailing white space in the query

Reply:
Fixed.